src/v/cluster/controller.cc

```diff
@@ -542,6 +546,22 @@ controller::start(cluster_discovery& discovery, ss::abort_source& shard0_as) {
           return _partition_balancer.invoke_on(
             partition_balancer_backend::shard,
             &partition_balancer_backend::start);
+      })
+      .then([this] {
+          auto bucket_opt = config::shard_local_cfg().cloud_storage_bucket();
```

Comment:
You should use `cloud_storage::configuration::get_bucket_config()` to make this work on ABS too.

Reply:
Done